Dockerfile

```diff
@@ -1,4 +1,20 @@
-FROM python:2.7.11-slim
-COPY src/main/python/* /
-ENTRYPOINT ["/bin/bash", "-c", "./runner.sh"]
+FROM python:2.7-slim as build_proto
+RUN  apt-get update \
+  && apt-get install -y wget \
+  && rm -rf /var/lib/apt/lists/*
+
+ADD  grpcio-tools.txt /
+RUN  ["pip","install","-r","grpcio-tools.txt"]
+
+RUN ["wget", "https://raw.githubusercontent.com/projectriff/function-proto/master/function.proto", "-P", "proto"]
```

Comment:
I'd rather the Dockerfile assumes this is already present. Fetching from `master` at build time means we have no control over exactly what winds up in the image and makes it harder to recreate a build.